src/sentry/static/sentry/app/components/forms/apiForm.tsx

```diff
@@ -40,14 +41,32 @@ export default class ApiForm extends Form<Props> {
     this.api.clear();
   }
 
+  getEnabledData() {
+    // Return a hash of data from non-disabled fields.
+    const data = {...this.state.data}; // copy to avoid mutating original
+    if (Array.isArray(this.props.children)) {
+      this.props.children.forEach(child => {
```

Comment:
FYI `for .. of` loops are more performant than `.forEach` loops.

Reply:
Done with ead6d23f6995a0511bf109d090c5eb19c0a72790.